plugins/rxp/test/RxpReaderTest.cpp

```diff
@@ -131,7 +130,7 @@ TEST(RxpReaderTest, testRead)
 TEST(RxpReaderTest, testNoPpsSync)
 {
     Options options = defaultRxpReaderOptions();
-    Option syncToPps("sync_to_pps", "false", "");
+    Option syncToPps("sync_to_pps", false);
```

Comment:
It's still the right way.  Booleans are weird.  There could be a bug.  I'll write a test.